app/views/layouts/_modal_dialog_download.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= stylesheet_link_tag('clickable') %>
```

Comment:
You can include this as part of the standard set of CSS includes for the application.